base/cvd/cuttlefish/host/commands/cvd/common_utils.cpp

```diff
@@ -168,7 +168,7 @@ std::string PerUserDir() {
 }
 
 std::string InstanceDatabasePath() {
-  return fmt::format("{}/instance_database.json", PerUserDir());
+  return fmt::format("{}/instance_database.pb", PerUserDir());
```

Comment:
nit: use `.textpb` extension https://protobuf.dev/reference/protobuf/textformat-spec/#text-format-files

Reply:
It's not text, it's binary.

Reply:
I see, suggestion for binary is `.binpb`

Reply:
done